src/app/shared/services/import-validation/import-validation.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { NO_LATIN_REGEX, RNOKPP_REGEX } from 'shared/constants/regex-constants';
+import { ImportEmployeesChosenRole } from 'shared/enum/enumUA/import-export';
+import { FieldValidationConfig, FieldsConfig } from 'shared/models/admin-import-export.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ImportValidationService {
+  constructor() {}
+  public checkForInvalidData(items: any[], config: FieldsConfig[]): void {
+    items.forEach((item) => {
+      item.errors = {};
+      config.forEach((field) => {
+        this.validateField(field.fieldName, item, field.validationParam);
+      });
+    });
+  }
+
+  private validateField(fieldName: string, item: any, config: FieldValidationConfig): void {
+    if (config.checkEmpty && !item[fieldName]) {
```

Comment:
I would prefer to update this method. Here are my suggestions: 
1. Use a validator map
```
{
  checkLanguage : (value) => !NO_LATIN_REGEX.test(value)
  ...
}
```
2. Create constants for magic numbers (2 and 50)
3. As I understand it, in the table we show only one error for one field - there are 2 ways -> show all errors or exit the method when at least one error is detected